content/zh/guide/v11/whats-new.md

```diff
@@ -0,0 +1,135 @@
+---
```

Comment:
This is referencing Preact 10, and we don't have a `whats-new.md` file in the English v11